### PR TITLE
Add release notes from the wiki

### PR DESCRIPTION
**Pull request by nickvergessen**
Copied from https://github.com/nextcloud/server/wiki/Nextcloud-10-Features

@MorrisJobke @blizzz @mar1u5

### CODE REVIEW

File: admin_manual/release_notes.rst [hunk 1]
```diff
@@ -6,4 +6,96 @@ Nextcloud |version| Release Notes
...
 -------------
 
-tbd
+General
+=======
+
+* Background jobs (cron) can now run in parallel `owncloud/core#24696 <https://github.com/owncloud/core/issues/24696>`_
+* Update notifications in client via API - You can now be notified in your desktop client about available updates for core and apps. The notifications are made available via the notifications API. `owncloud/core#24444 <https://github.com/owncloud/core/pull/24444>`_
+* Multi bucket support for primary objectstore integration `owncloud/core#24760 <https://github.com/owncloud/core/pull/24760>`_
+* `Automated Tagging of files <https://github.com/owncloud/automated_tagging>`_: An app for Nextcloud that automatically assigns tags to newly uploaded files based on some conditions
```
Comment (MorrisJobke) [inline admin_manual/release_notes.rst]: owncloud?

**Comment by MorrisJobke** [comment]
Beside that :+1:

**Comment by mar1u5** [comment]
Besides Branding Stuff :+1: ... But I think this is true anyway cause it references to Feature PRs we dont have over here in Nextcloud :grin: @MorrisJobke

**Comment by nickvergessen** [comment]
Well this one app link is wrong ;)

**Comment by nickvergessen** [comment]
Fixed

**Pull request merged**